_posts/2020-4-19-pytorch-library-updates-new-model-serving-library.md

```diff
@@ -0,0 +1,82 @@
+
+See the full torch_xla release notes [here](https://github.com/pytorch/xla/releases). Full docs and tutorials can be found [here](https://pytorch.org/xla/) and [here](https://cloud.google.com/tpu/docs/tutorials).
+
+## PyTorch Domain Libraries
+
+torchvision, torchtext, and torchaudio complement PyTorch with common datasets, models, and transforms in each domain area. We’re excited to share new releases for all three domain libraries alongside PyTorch 1.5 and the rest of the library updates. For this release, all three domain libraries are removing support for Python2 and will support Python3 only.
+
+### torchaudio 0.5
+The torchaudio 0.5 release includes new transforms, functionals, and datasets. Highlights for the release include:
+
+* Added the Griffin-Lim functional and transform, `InverseMelScale` and `Vol` transforms, and `DB_to_amplitude`. 
+* Added support for `allpass`, `fade`, `bandpass`, `bandreject`, `band`, `treble`, `deemph`, and `riaa` filters and transformations.
+* New datasets added including `LJSpeech` and `SpeechCommands` datasets. 
+
+See the release full notes [here](https://github.com/pytorch/audio/releases) and full docs can be found [here](https://pytorch.org/audio/).
+
+### torchvision 0.6
+The torchvision 0.6 release includes updates to datasets, models and a significant number of bug fixes. Highlights include:
+
+* Faster R-CNN now supports negative samples which allows the feeding of images without annotations at training time.
+* Added `aligned` flag to `RoIAlign` to match Detectron2. 
+* Refactored abstractions for C++ video decoder
+
+See the release full notes [here](https://github.com/pytorch/vision/releases) and full docs can be found [here](https://pytorch.org/docs/stable/torchvision/index.html).
+
+### torchtext 0.6
+The torchtext 0.6 release includes a number of bug fixes, improvements to documentation and updates to the dataset abstraction. Highlights for the release include:
+
+* Fixed an issue related to the SentencePiece dependency in conda package.
+* Added support for the experimental IMDB dataset to allow a custom vocab.
+* A number of documentation updates including adding a code of conduct and a deduplication of the docs on the torchtext site. 
+
+Your feedback and discussions on the experimental datasets API are welcomed. You can send them to [issue #664](https://github.com/pytorch/text/issues/664). We would also like to highlight the pull request [here](https://github.com/pytorch/text/pull/701) where the latest dataset abstraction is applied to the text classification datasets. The feedback can be beneficial to finalizing this abstraction. 
```

Comment:
I'd add something along the lines of "Based on user's feedback torchtext's dataset abstractions are currently being redesigned." to the beginning of this line.